build/Dockerfile

```diff
@@ -10,7 +10,7 @@ COPY . /go/src/github.com/nginxinc/nginx-kubernetes-gateway
 RUN make build
 
 FROM alpine:3.18 as capabilizer
-RUN apk add --no-cache libcap
+RUN apk update && apk add --no-cache libcap
```

Comment:
This is an intermediate container that is never shipped and we only use `libcap` from there. Do we really need to update all the other packages?

Reply:
This is ensuring that we use the latest version of libcap. We had an issue in the nginx container where installing libcap from that version of alpine was outdated and had CVEs.

Reply:
I don't think this will fix it...`apk add --no-cache libcap` always installs the latest version of `libcap` already and if `alpine:3.18` doesn't change this layer will be cached anyway.

In the CI we always build the image without cache in `main`, so the latest version of `libcap` will always be installed.

Reply:
Oh I see. I can close this then if it's a no-op.